scripts/jenkins/qa_openstack.sh

```diff
@@ -473,6 +473,9 @@ id-3fb3ca3a-8e3a-4565-ba73-16413d445e25
 __EOF__
         blacklistoptions="--blacklist-file tempest-blacklist.txt"
         ;;
+    openstackmaster)
+        blacklistoptions="--black-regex negative"
```

Comment:
Can we just disable the dynamic routing tests with `^neutron_tempest_plugin.neutron_dynamic_routing` ?

Reply:
we can also do that... is that preferred?

Reply:
I would prefer this. Imo it's more clean than what is failing (and sorry for the late response)